views/demo.pug

```diff
@@ -4,13 +4,13 @@ block content
 	.container-fluid
 		.row
 			.col-sm-3.col-lg-2
-				nav.side-navbar.navbar-default.navbar-fixed-side
+				.side-navbar
 					ul
-						li
-							a(href="#") Link-1
-						li
-							a(href="#") Link-2
-						li
-							a(href="#") Link-3
+						a(href="#")
+							li Link-1
+						a(href="#")
```

Comment:
I will build the sidebar for you properly as an exp to help you better understand when I get a chance